docs/core/extensions/logger-message-generator.md

```diff
@@ -111,11 +111,12 @@ Consider the example logging output when using the `JsonConsole` formatter.
 
 When using the `LoggerMessageAttribute` on logging methods, there are some constraints that must be followed:
 
-- Logging methods must be `static`, `partial`, and return `void`.
+- Logging methods must be `partial` and return `void`.
 - Logging method names must *not* start with an underscore.
```

Comment:
It seems that also parameters can't start with an underscore.

https://github.com/dotnet/runtime/blob/2330b4bef645bf8bcd3011f35968e55192c697f5/src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Parser.cs#L387-L392

Reply:
That's on the next line :)